starter-code/views/about.hbs

```diff
@@ -0,0 +1,12 @@
+<div id="text-hide">
+    <h2>Andy TISBA</h2>
+    <p>Mail: <strong><a href='mailto:andy.tisba@gmail.com'>andy.tisba@gmail.com</a></strong></p>
+    <p>Phone: <strong>+33609471190</strong></p>
```

Comment:
Don't ever put your phone number / email on a website !  Crawlers love thse ! 
You will end up with spam / shady phone calls 😆